src/bundle/Resources/views/themes/admin/account/forgot_password/index.html.twig

```diff
@@ -14,12 +14,14 @@
         <p>{{ 'ezplatform.forgot_password.reset_your_password.reason.migration'|trans|desc('Your password has expired, change it.') }}</p>
     {% endif %}
     {% if form_forgot_user_password is defined %}
-        {{ form_start(form_forgot_user_password, {'attr': {'class': 'ibexa-form-validate ibexa-login__forgot-password-form'}}) }}
+        {{ form_start(form_forgot_user_password, {'attr': {'class': 'ibexa-form-validate ibexa-login__forgot-password-form'} }) }}
```

Comment:
We usually avoid formatting unrelated code in bug fixes, because in such cases it may be harder to merge it up. And I think that if we want to format it, we should format it completely, so:
```suggestion
        {{ form_start(form_forgot_user_password, { attr: { class: 'ibexa-form-validate ibexa-login__forgot-password-form } }) }}
```